src/svg.js

```diff
@@ -42,7 +42,7 @@ Canvas.prototype.createSubCanvas = function() {
   return new Canvas(group, this.defs);
 };
 
-Canvas.init = elmOrId => {
+Canvas.init = function(elmOrId, _width = '100%', _height = '100%') {
```

Comment:
nit: no need for `_` here. You can call these `width` and `height`.

Reply:
also please use `=>` syntax instead of `function` unless there's a reason `=>` can't be used.

Reply:
If use width directly, yarn build shows:"error  Expected property shorthand  object-shorthand"